source/guide_mautic.rst

```diff
@@ -0,0 +1,381 @@
+
+2. Let Apache handle the backend for your added domain.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ uberspace web backend set mautic.mysite.com --apache
+  Set backend for mautic.mysite.com/ to apache.
+  [isabell@stardust ~]$
+
+
+Step 2: Create the database
+---------------------------
+
+.. include:: includes/my-print-defaults.rst
+
+Create a new MySQL database for Mautic.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mysql -e "CREATE DATABASE ${USER}_mautic;"
+  [isabell@stardust ~]$
+
+
+Installation (using Composer)
+=============================
+
+Change to Apache's :manual:`DocumentRoot <web-documentroot>` directory:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ cd /var/www/virtual/$USER
+  [isabell@stardust isabell]$
+
+Step 1: Download Mautic
+------------------------
+
+Download your Mautic instance into it's own subdirectory:
+
+.. code-block:: console
+
+  [isabell@stardust isabell]$ composer create-project mautic/recommended-project:^4 mautic --no-interaction
+  Creating a "mautic/recommended-project:^4" project at "./mautic"
+  Installing mautic/recommended-project (4.4.7)
+    - Installing mautic/recommended-project (4.4.7): Extracting archive
+  Created project in /var/www/virtual/isabell/mautic
+  Loading composer repositories with package information
+  Updating dependencies
+  Lock file operations: 231 installs, 0 updates, 0 removalss
+  - Locking aws/aws-crt-php (v1.2.1)
+  [...]
+  - Locking willdurand/negotiation (3.1.0)
+  Writing lock file
+  Installing dependencies from lock file (including require-dev)
+  Package operations: 231 installs, 0 updates, 0 removals
+    - Syncing friendsofsymfony/oauth-server-bundle (dev-doctrine-fix 33bc2f4) into cache
+    - Installing composer/installers (v1.12.0): Extracting archive
+  [...]
+  Generating autoload files
+  110 packages you are using are looking for funding.
+  Use the `composer fund` command to find out more!
+  Symfony recipes are disabled: "symfony/flex" not found in the root composer.json
+
+  Package {VENDOR/PACKAGE} is abandoned, you should avoid using it. Use {VENDOR/PACKAGE} instead. # Might come up multiple times. You can safely ignore it.
+
+  Loading composer repositories with package information
+  Updating dependencies
+  Nothing to modify in lock file
+  Installing dependencies from lock file (including require-dev)
+  Nothing to install, update or remove
+  Generating autoload files
+  Symfony recipes are disabled: "symfony/flex" not found in the root composer.json
+
+  Scaffolding files for mautic/core-lib:
+     - Copy [project-root]/.env.dist from assets/scaffold/files/.env.dist
+    [...]
+  No security vulnerability advisories found
+  Project configuration is disabled: "symfony/flex" not found in the root composer.json
+
+  Congratulations, you’ve installed the Mautic codebase
+  from the mautic/recommended-project template!
```

Comment:
reduce the output here a lot, just keep these two lines with "congratulations youve installed..." and set a `[...]` before and after